packages/yoastseo/spec/scoring/productPages/seoAssessorSpec.js

```diff
@@ -242,20 +242,34 @@ describe( "running assessments in the product page SEO assessor", function() {
 		] );
 	} );
 
-	it( "additionally runs assessments that require a product with variants or a price", function() {
+	it( "additionally runs assessments that require the SKU to be detectable, and that shouldn't be applicable if the" +
+		"product has variants and we don't want to assess variants", function() {
 		const customData = {
+			canRetrieveGlobalSku: true,
+			canRetrieveVariantSkus: true,
 			hasVariants: true,
-			hasPrice: false,
-			productType: "simple",
+			productType: "variable",
 		};
 		assessor.assess( new Paper( "", { customData } ) );
 		const AssessmentResults = assessor.getValidResults();
 		const assessments = getResults( AssessmentResults );
 
-		expect( assessments ).toContain( "productIdentifier" );
 		expect( assessments ).toContain( "productSKU" );
 	} );
 
+	it( "additionally runs assessments that shouldn't be applicable if the product has variants and we don't want" +
+		" to assess variants", function() {
```

Comment:
This description is hard to read. i would suggest something easier along the lines of:

```suggestion
	it( "runs the productIdentifierAssessment when it is applicable (it has variants and assessVariants is True)", function() {
```